UT-Road_1/UT_Road_1.ifc

```diff
@@ -0,0 +1,105 @@
+/* time_stamp */ '2021-01-12T16:51:14',
+/* author */ ('stefan'),
+/* organization */ ('TUM Chair of Computational Modeling and Simulation'),
+/* preprocessor_version */ 'GeometryGymIFC v0.1.6.0 by Geometry Gym Pty Ltd built 2021-01-12T15:50:49',
+/* originating_system */ 'GeometryGymIFC v0.1.6.0',
+/* authorization */ 'None');
+
+FILE_SCHEMA (('IFC4X3_RC2'));
+ENDSEC;
+
+DATA;
+#1= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#2= IFCAXIS2PLACEMENT3D(#1,$,$);
+#3= IFCLOCALPLACEMENT($,#2);
+#4= IFCSITE('3gqVKFPcHBbhBOQxCsxlUl',#10,'sampleSite',$,$,#3,$,$,$,$,$,$,$,$);
+#5= IFCPERSON('stefan','stefan',$,$,$,$,$,$);
+#6= IFCORGANIZATION($,'Unknown',$,$,$);
+#7= IFCPERSONANDORGANIZATION(#5,#6,$);
+#8= IFCAPPLICATION(#9,'0.1.6.0','GeometryGymIFC v0.1.6.0','GeometryGymIFC v0.1.6.0');
+#9= IFCORGANIZATION($,'Geometry Gym Pty Ltd',$,$,$);
+#10= IFCOWNERHISTORY(#7,#8,$,.ADDED.,1610466672,$,$,1610466672);
+#11= IFCPROJECT('3sq579Y5HAUu6Kn$aEETIo',#10,'sampleProject',$,$,$,$,(#38),#12);
+#12= IFCUNITASSIGNMENT((#13,#14,#15,#16,#17));
+#13= IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#14= IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#15= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#16= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#17= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#18= IFCRELAGGREGATES('33axnoY2f379TuHe63A5XU',#10,$,$,#11,(#4));
+#19= IFCCARTESIANPOINT((0.0,0.0));
+#20= IFCCARTESIANPOINT((0.0,0.0));
+#21= IFCAXIS2PLACEMENT2D(#20,$);
+#22= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#23= IFCALIGNMENT('3K9hXoZO93PfAKgAT0euP3',#10,'DefaultAlignment',$,$,#26,$,$,$);
+#24= IFCRELCONTAINEDINSPATIALSTRUCTURE('2vg6zY$hPFvgkKx2jxox0T',#10,'Site','Site Container for Elements',(#23),#4);
+#25= IFCAXIS2PLACEMENT3D(#22,$,$);
+#26= IFCLOCALPLACEMENT($,#25);
+#27= IFCCARTESIANPOINT((-29.29264,-6.90761));
+#28= IFCALIGNMENTHORIZONTALSEGMENT($,$,#27,0.462753315190078,0.0,0.0,72.3546021022319,$,.LINE.);
+#29= IFCLOCALPLACEMENT($,#21);
+#30= IFCALIGNMENTHORIZONTAL('0d38x89KL2S9T71tkxjwYF',#10,$,$,$,#29,#44,0.0,(#28));
+#31= IFCAXIS2PLACEMENT2D(#27,#32);
+#32= IFCDIRECTION((0.89482677,0.44641354));
+#33= IFCDIRECTION((1.0,0.0));
+#34= IFCVECTOR(#33,1.0);
+#35= IFCLINE(#19,#34);
+#36= IFCCURVESEGMENT(.DISCONTINUOUS.,#31,IFCNONNEGATIVELENGTHMEASURE(0.0),IFCNONNEGATIVELENGTHMEASURE(72.3546021022319),#35);
+#37= IFCCOMPOSITECURVE((#36),.F.);
+#38= IFCGEOMETRICREPRESENTATIONCONTEXT($,'Model',3,0.0001,#40,#41);
+#39= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#40= IFCAXIS2PLACEMENT3D(#39,$,$);
+#41= IFCDIRECTION((0.0,1.0));
+#42= IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Axis','Model',*,*,*,*,#38,$,.GRAPH_VIEW.,$);
+#43= IFCSHAPEREPRESENTATION(#42,'Axis','Curve',(#37));
+#44= IFCPRODUCTDEFINITIONSHAPE($,$,(#43));
+#45= IFCRELAGGREGATES('3HWOHIGon3oRsTCDQ5$3to',#10,$,$,#23,(#30));
+#46= IFCALIGNMENTSEGMENT('1qVGOURzHFqvsVaIYUnUDP',#10,$,$,$,#29,#49,#28);
+#47= IFCRELNESTS('0BPOUVsET2AwrPtwy3Fw3B',#10,$,$,#30,(#46));
+#48= IFCSHAPEREPRESENTATION(#42,'Axis','Curve2D',(#36));
+#49= IFCPRODUCTDEFINITIONSHAPE($,$,(#48));
```

Comment:
@SergejMuhic: `#46` has `#36` as one of its representations. It is not an `IfcCurve` though. So how to denote that in `#48`?

*Note: I'm selecting all the in-between lines, you can safely ignore them.*